Code/Source/VehicleDynamics/DriveModels/SimplifiedDriveModel.cpp

```diff
@@ -24,7 +24,11 @@ namespace VehicleDynamics
             serialize->Class<SimplifiedDriveModel, DriveModel>()
                 ->Version(1)
                 ->Field("SteeringPID", &SimplifiedDriveModel::m_steeringPid)
-                ->Field("SpeedPID", &SimplifiedDriveModel::m_speedPid);
+                ->Field("SpeedPID", &SimplifiedDriveModel::m_speedPid)
+                ->Field("MaxSpeedImpulse", &SimplifiedDriveModel::maxSpeedImpulse)
```

Comment:
This is something that always needs tuning? There is no good default or formula based on radius, mass?

Reply:
Also, do we need to limit steering as well? If so, then adding maxCommandValue to PIDConfiguration instead would be better.

Reply:
I believe we can find (and test) a formula for this, but IMO manual tuning will be still useful. Maybe we should place settings like this in "deeper" parts of the config tree, something like `Advanced settings`.

As for the steering force limit: currently, it seems to be working fine without it, but there is a chance (or risk) that we'll be forced to add it. However, these limits may have different values, so we can't use just one value.

Reply:
If moved to PIDConfiguration, these would be two values, defaults could be overwritten in the SimplifiedDriveModel when constructing these. But for now we can leave it as it is with one value if the second one would not be used.

Reply:
Good point. Ok, let's leave it.